controllers/nodenetworkstate_controller.go

```diff
@@ -102,3 +104,17 @@ func (r *NodeNetworkStateReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		WithEventFilter(onDeleteForThisNode).
 		Complete(r)
 }
+
+func isForceRefresh(updateEvent event.UpdateEvent) bool {
```

Comment:
Just an idea, what do you think about calling this `shouldForceRefresh`, as there is additional logic and not just simple label check

Reply:
Good idea, I will change it after fixing flakiness at test.